e2e/setup/docker_commander.go

```diff
@@ -85,12 +85,12 @@ func StartDockerCommander(opts StartOptions) (*DockerCommander, error) {
 			Mounts: []mount.Mount{
 				{
 					Type:   mount.TypeBind,
-					Source: path.Join(utils.GetProjectRoot(), "e2e-data"),
+					Source: path.Join(".", "e2e-data"),
```

Comment:
Does that get resolved to "${cwd}/e2e-data"? Can we write is as `filepath.Abs("./e2e-data")`?

Reply:
It does. We can write it as `filepath.Abs`, but it returns `path, err, so the code would be less readable.  The `path.Join(".", ...` happens in several other places too, so I'm inclined to keep it as is. We can change it in a follow-up PR if desirable!